content/documentation/staging/quick-start/index.adoc

```diff
@@ -16,108 +16,52 @@ weight: 2
 :imagesdir: /images/quickinstall/
 :sectlinks:
 
-icon:lightbulb[size=1x]{nbsp} This page is intended for users installing Kiali for the first time, not recommended for custom production environments. In case of custom installations, visit the link:{{< ref "/documentation/staging/installation-guide" >}}[installation guide].
+icon:lightbulb[size=1x]{nbsp} This page is intended for users installing Kiali for the first time. These instructions are not recommended to install Kiali in production environments. In the case of custom installations, visit the link:{{< ref "/documentation/staging/installation-guide" >}}[installation guide].
 
-== Installing Kiali
+== Install via Kiali Server Helm Chart
```

Comment:
I think there needed to be a reference to the prerequistes section and also the OpenShift prep note before proceeding.  User should go there, make sure everything is kosher, and then return.

Reply:
done